client/src/lib/slate-pointers/ShowExpandedPointer.tsx

```diff
@@ -57,29 +57,40 @@ export class ShowExpandedPointer extends React.Component<any, any> {
   }
 
   public render() {
+    const isNestedAsExportingPointer = this.props.availablePointers.some(pointer => {
+      return pointer.nodes.some(node => {
+        return (node.type === "pointerExport" && node.data.pointerId === this.props.exportingPointer.data.pointerId);
+      });
+    });
+
     return (
-      <div>
+      <div
+        style={{
+          paddingLeft: isNestedAsExportingPointer ? "5px" : 0,
+        }}
+      >
         {this.props.exportingPointer.nodes.map((node, index) => {
           if (
             node.object === "inline" ||
             node.object === "GeneratedNestedExportNode"
           ) {
             return (
-              <InlineNode
-                node={node}
-                key={index}
-                blockEditor={this.props.blockEditor}
-                availablePointers={this.props.availablePointers}
-                pointerId={this.props.exportingPointer.pointerId}
-                onMouseOver={this.props.onMouseOverPointerImport}
-                isHoverable={this.props.isHoverable}
-              />
+              <span key={index} style={{ marginLeft: "4px", marginRight: "6px" }}>
```

Comment:
Use styled-components for consistency?

Reply:
I think styled-components make sense when you're reusing the styles, or when you need something like a selector or pseudo-selector, or when you're using a newish feature and want automatic vendor prefixing, but otherwise I find it makes files harder to parse. When the style information is more closely coupled to the markup structure, I have a better feel for the CSS structure (what styles are nested in what) and an easier time making quick changes. I think this is especially true when the styling is just a few pixels of padding or margin.

Reply:
Sounds reasonable.

Reply:
I think I'm pretty ok with this; in cases where there are only a few position adjustments, splitting it out can be a bit of a pain.